app/views/users/new.html.erb

```diff
@@ -1,7 +1,23 @@
-<% provide(:title, t("title.signup")) %>
-<h1>
-  <%= t("user.new.title")%>
-</h1>
-<p>
-  <%= t("user.new.signup_quote") %>
-</p>
+<% provide :title, t("title.signup") %>
+<h1><%= t("title.signup") %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(@user) do |f| %>
+      <%= render "shared/error_messages" %>
+
+      <%= f.label :name, t("user.new.label_name")%>
```

Comment:
```suggestion
      <%= f.label :name, t "user.new.label_name" %>
```
sau `t` ko còn các thành phần khác nên em lượt bỏ ngoặc tròn dc

ngoài ra, em có thể đọc [link này](https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models), I18n của Rails có hỗ trợ translation với Active record  đó bé